script/1_0_2current_en_script/Common/00_00.txt

```diff
@@ -137,90 +137,90 @@ Hey, why does the hospital have cracks everywhere?"
-But I can see it.
-And it scares me so much, I can't even walk.
+No one could see it.
+I guessed that was how they could remain so calm.
+They didn't even notice how messed up this world was, at all.
+They didn't know their everyday life was full of holes.
+But I could see it.
+And it scared me so much, I couldn't even walk.
 <Page51>
-Speaking of which, my ears have been frozen since then, and it's gotten quieter despite the noise.
-Only the wound in my chest reverberates like a siren.
+Speaking of which, my ears had been frozen since then, and it had gotten quieter despite the noise.
+Only the wound in my chest reverberated like a siren.
 But "when" was "that"?
-I try to recall, but it's no use. I scratch my head, but even that doesn't help. I'm sure I must've forgotten about it because it wasn't very important. Like a faucet that kept spilling water no matter how hard you close it.
+I tried to recall, but it was no use. I scratched my head, but even that didn't help. I was sure I had forgotten about it because it wasn't very important. Like a faucet that kept spilling water no matter how hard you close it.
 <Page52>
 ......Ah, I see.
-That explains everything.
-It makes sense there isn't a mirror here.
+That explained everything.
+It made sense there wasn't a mirror here.
 If that were the case, I would have ■ before anything else.
-I'm sure I'd crack up laughing.
-In other words, it's not the world around me that's breaking down.
-I'm the only one that's broken.
+I was sure I would have been cracking up laughing.
+In other words, it wasn't not the world around me that was breaking down.
+I was the only one that was broken.
 <Page53>
-That's probably why.
-It's been two weeks since then and no one believes me.
-It's been two weeks since then and no one's come to see me.
-It's been two weeks.
-I am alone, living in a world full of cracks----
+That was probably why.
+It had been two weeks since then and no one believed me.
+It had been two weeks since then and no one had come to see me.
+It had been two weeks.
+I was alone, living in a world full of cracks----
 <Page54>
-I don't want to be in a hospital room.
-I don't want to be in a place full of lines.
-I get away from here,
+I didn't want to be in a hospital room.
+I didn't want to be in a place full of lines.
+I got away from here,
 and decided to go to a faraway place with no one around.
 <Page55>
-I'm not the one who's crazy.
+I was not the one who was crazy.
 I jumped out of the window as if to convince myself that there was something wrong with that place.
 <Page56>
 But the wound in my chest was so painful that I could only run a little.
-When I came to my senses.
+When I came back to my senses.
 I found myself in a field next to the hospital.
 I hadn't gotten very far.
 <Page57>
 "......Gohh."
-My chest hurts and I feel so sad that I crouched down on the ground, coughing.
+My chest hurt and I felt so sad that I crouched down on the ground, coughing.
```

Comment:
Line above this is still Gohh onomatopoeia for cough, but it is changed a couple lines below - standardize?